packages/entity-server/src/routes/hierarchy/relations/EntityRelationsRoute.ts

```diff
@@ -4,122 +4,15 @@
  */
 
 import { Route } from '@tupaia/server-boilerplate';
-import { reduceToDictionary, reduceToArrayDictionary } from '@tupaia/utils';
-import { EntityType } from '../../../models';
-import { formatEntitiesForResponse } from '../format';
 import { RelationsRequest } from './types';
-
-type Pair = {
-  descendant: string;
-  ancestor: string;
-};
+import { RelationsResponseBuilder } from './RelationsResponseBuilder';
 
 export class EntityRelationsRoute extends Route<RelationsRequest> {
-  async buildAncestorCodesAndPairs(descendants: EntityType[]): Promise<[string[], Pair[]]> {
```

Comment:
All logic moved to `packages/entity-server/src/routes/hierarchy/relations/RelationsResponseBuilder.ts` so as to be re-usable with `MultiEntityRelationsRoute`

Originally I tried to have the multi entity route call this route directly, however that was proving difficult for typescript and generally I found it a bit of an ugly approach...